src/Core/Services/OpenAPI/OpenApiDocumentor.cs

```diff
@@ -56,13 +57,22 @@ public class OpenApiDocumentor : IOpenApiDocumentor
         public const string DOCUMENT_CREATION_UNSUPPORTED_ERROR = "OpenAPI description document can't be created when the REST endpoint is disabled globally.";
         public const string DOCUMENT_CREATION_FAILED_ERROR = "OpenAPI description document creation failed";
 
+        public void Documentor_ConfigChangeEventReceived(object? sender, CustomEventArgs args)
```

Comment:
nit: why the `_` in the name? As per our naming convention, wouldnt this be `DocumentorConfigChangeEventReceived`? Same for DocumentorSubscribe